### PR TITLE
add missing rbac for persistentvolume during deletion

### PR DESCRIPTION
**Pull request by leelavg**
cherrypick of https://github.com/ceph/ceph-csi-operator/pull/141 and bundle generation

### CODE REVIEW

File: bundle/manifests/cephcsi-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -733,7 +735,7 @@ spec:
                       fieldPath: metadata.namespace
                 - name: CSI_SERVICE_ACCOUNT_PREFIX
                   value: ceph-csi-
-                image: quay.io/ocs-dev/cephcsi-operator:latest
+                image: quay.io/cephcsi/ceph-csi-operator:latest
```
Comment (Madhu-1) [inline bundle/manifests/cephcsi-operator.clusterserviceversion.yaml:738]: @leelavg we need to revert this back

Comment (leelavg) [reply bundle/manifests/cephcsi-operator.clusterserviceversion.yaml:738]: yes, I'm checking why it got changed.

Comment (leelavg) [reply bundle/manifests/cephcsi-operator.clusterserviceversion.yaml]: done.

**Review by Madhu-1 — APPROVED** [review]

File: Makefile.Downstream.mk [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# defining variables here before including "Makefile" makes these variables unique for current makefile
+IMAGE_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
+IMAGE_TAG ?= latest
+IMAGE_NAME ?= cephcsi-operator
+
```
Comment (Madhu-1) [inline Makefile.Downstream.mk:6]: @iamniting PTAL

**Comment by leelavg** [comment]
/cherrypick release-4.17
https://bugzilla.redhat.com/show_bug.cgi?id=2313203

File: Makefile.Downstream.mk (hunk 2 above)
Comment (iamniting) [reply Makefile.Downstream.mk:6]: We need to keep in mind to make changes here if we make any changes in the actual makefile.

**Review by iamniting — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
Please don't do manual cherry-pick of commits from upstream to downstream main branch.
The sync pipeline runs daily and syncs the commits. If anything urgent is needed, feel free to ping me to run the pipeline manually.

**Comment by leelavg** [comment]
> Please don't do manual cherry-pick of commits from upstream to downstream main branch.
> The sync pipeline runs daily and syncs the commits. If anything urgent is needed, feel free to ping me to run the pipeline manually.
- cool, didn't know that, will take care. Thanks.

**Pull request merged**